yarn.lock

```diff
@@ -3272,6 +3279,13 @@ opn@^5.3.0:
   dependencies:
     is-wsl "^1.1.0"
 
+optimist@~0.3.5:
+  version "0.3.7"
+  resolved "https://registry.yarnpkg.com/optimist/-/optimist-0.3.7.tgz#c90941ad59e4273328923074d2cf2e7cbc6ec0d9"
+  integrity sha1-yQlBrVnkJzMokjB00s8ufLxuwNk=
+  dependencies:
+    wordwrap "~0.0.2"
+
```

Comment:
I'm not sure why this optimist block and the wordwrap one below are appearing. Any ideas on how these got added and if I should remove them?

Reply:
If `yarn.lock` was generated this way, I suppose these could be `checksum`'s dependencies.